launch/tier4_planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py

```diff
@@ -342,12 +398,16 @@ def launch_setup(context, *args, **kwargs):
                 "launch_detection_area": True,
                 "launch_virtual_traffic_light": True,
                 "launch_occlusion_spot": True,
+                "launch_dynamic_obstacle_stop": False,
                 "launch_no_stopping_area": True,
                 "forward_path_length": 1000.0,
                 "backward_path_length": 5.0,
                 "max_accel": -2.8,
                 "delay_response_time": 1.3,
             },
+            common_param,
+            base_param,
```

Comment:
what is the difference `base` and `common` param?

Reply:
They are required to use motion velocity smoother in this module.
base param has parameters required by the smoother node.
common param has parameters required by several planning nodes.

https://github.com/autowarefoundation/autoware.universe/blob/main/launch/tier4_planning_launch/config/scenario_planning/common/motion_velocity_smoother/motion_velocity_smoother.param.yaml
https://github.com/autowarefoundation/autoware.universe/blob/main/launch/tier4_planning_launch/config/scenario_planning/common/common.param.yaml

Reply:
I understood about the difference between base and common.
But what is the difference between base and smoother?
![image](https://user-images.githubusercontent.com/8327598/170437438-e0bd2406-4c5c-409a-979b-c54a25b43194.png)

Reply:
smoother param is parameters for specific smoother algorithm (Analytical, JerkFiltered, L2, Linf)
https://github.com/autowarefoundation/autoware.universe/tree/main/launch/tier4_planning_launch/config/scenario_planning/common/motion_velocity_smoother
base_param is common parameter for all smoother algorithms.

Reply:
By the way, I noticed parameter path for `smoother_param` should be passed by `scenario_planning.launch.xml`  to use the same smoother algorithm. (Currently, Analytical.param is specified in `behavior_planning.launch.py`)

Can I change the launch to pass argument of `smoother_type` to behavior_velocity_planner? cc: @taikitanaka3 
https://github.com/autowarefoundation/autoware.universe/blob/main/launch/tier4_planning_launch/launch/scenario_planning/scenario_planning.launch.xml#L26-L27

Reply:
base param name may mislead it means base parameter of behavior velocity because there is smoother param.
It is better to merge base param and smoother param. Is it possible?

Reply:
> merge base param and smoother param

Does this mean merge base parameter file and smoother parameter file into 1 file?
I think it would be difficult since motion_velocity_smoother also use them.

How about simply change the name of `base_param` to `smoother_base_param` in this launch file?

Reply:
@yukkysaito 
I renamed in https://github.com/autowarefoundation/autoware.universe/pull/752/commits/cc29662c1f5104fc9d6e26b81560aacb3986fc43.


> By the way, I noticed parameter path for smoother_param should be passed by scenario_planning.launch.xml to use the same smoother algorithm in motion_velocity_smoother and behavior_velocity_planner. (Currently, Analytical.param is specified in behavior_planning.launch.py)

As for this, I will create an issue and discuss it there.